src/it/java/com/google/checkstyle/test/chapter4formatting/rule412nonemptyblocks/RightCurlyTest.java

```diff
@@ -37,15 +37,22 @@ protected String getPackageLocation() {
         return "com/google/checkstyle/test/chapter4formatting/rule412nonemptyblocks";
     }
 
+    private Configuration getModuleConfigs() {
+        return createTreeWalkerConfig(
+                getModuleConfigsByIds("RightCurlySame", "RightCurlyAlone"));
```

Comment:
it not clear how it will load
https://github.com/checkstyle/checkstyle/blob/2d4796b5f4996f901afa930776c001472f585230/src/main/resources/google_checks.xml#L98-L104
we need it.

Reply:
It is loaded by id. It is called `RightCurlyAlone`.

Reply:
Is this suppression loaded?

Reply:
2 IDs are specified, the code returns 3 configurations. So yes. 1 module has the id `RightCurlySame`. 2 modules have the id `RightCurlyAlone`.